### DISCUSSION TITLE
求教overleaf该如何使用西电研究生毕业论文模板，我阅读完了文档还是不知道怎么做，希望有人能告诉我

### DISCUSSION THREAD

**Discussion by ppppls115** [note286/xduts#168] [Q&A] [answered]
求教overleaf该如何使用西电研究生毕业论文模板，我阅读完了文档还是不知道怎么做，希望有人能告诉我

**Comment by note286** [comment, marked as answer]
根据readme，下载`.cls`文件，上传至Overleaf，然后编写`.tex`文件，修改编译器为XeLaTeX，编译即可。

**Reply by ppppls115** [reply]
感谢，刚才发现是编译器没改。之前一直用的作者您停更的那个项目，用起来感觉没啥大问题，但是老师说公式得用新罗马体。我也不知道老项目的怎么换，就过来改成最新版的了！

**Reply by pyh-129** [reply]
请问为何在overleaf更换字体后找不到字体：
```Package fontspec Error: The font "SimSun" cannot be found.```

**Reply by note286** [reply]
@pyh-129 你可能需要设置`font-type=file`，并设置`font-path`，详情请查阅手册。另外，不是很建议学位论文使用Overleaf，后期可能会遇到编译超时的问题。

**Reply by pyh-129** [reply]
谢谢，我是设置了路径后还是无法找到：请问这样设置不正确吗
```\xdusetup{
     style = {     
        font-type = file,
        font-path = {tex/xelatex/xduts/xdufont.sty}, 
        cjk-font = fandol,
        latin-font =tacn,
        before-skip = {24pt,21pt,18pt,12pt,12pt,12pt},
        after-skip = {18pt,12pt,12pt,12pt,12pt,12pt},
        subsec-zihao=-4,
        math-font=xits,
        },
}
```

**Reply by note286** [reply]
@pyh-129 当然不对，`xdufont`是单独的宏包，与`xduugthesis`无关，`font-path`是指定你字体所在的文件夹。你可以参考一下 https://github.com/note286/xduts/discussions/89 。

**Reply by peachk1024** [reply]
需要上传哪些文件到overleaf，具体文件名是啥

**Reply by note286** [reply]
@peachk1024 目前Overleaf已经内置xduts，只需要根据手册使用即可，除自定义字体文件外无需上传。
